Webpack.NET.Tests/TestConfigurationExtensions.cs

```diff
@@ -49,8 +49,9 @@ public void GetWebpack_Throws_On_Null_Application()
         public void GetWebpack_Throws_On_Missing_Webpack()
         {
             var application = new Mock<HttpApplicationStateBase>();
-            var error = Assert.Throws<ApplicationException>(() => application.Object.GetWebpack());
-            Assert.That(error.Message, Is.EqualTo("Webpack has not been configured, have you called HttpApplication.ConfigureWebpack()?"));
+            var error = Assert.Throws<InvalidOperationException>(() => application.Object.GetWebpack());
+            // Message could be localized, so do not check
+            //Assert.That(error.Message, Is.EqualTo("Webpack has not been configured, have you called HttpApplication.ConfigureWebpack()?"));
```

Comment:
If we're not going to check it then I'd suggest removing the assertion.  My feeling is that it's better to have some check on the message (to confirm it's a useful one rather than something meaningless) and worry about localisation at a later date!